plugin/src/main/java/io/jenkins/plugins/coverage/metrics/model/Baseline.java

```diff
@@ -27,23 +27,23 @@ public enum Baseline {
      * Coverage of the modified lines (e.g., within the modified lines of a pull or merge request) will focus on new or
      * modified code only.
      */
-    MODIFIED_LINES(Messages._Baseline_MODIFIED_LINES(), "changeCoverage", CoverageLevel::getDisplayColorsOfCoverageLevel),
+    MODIFIED_LINES(Messages._Baseline_MODIFIED_LINES(), "modifiedLinesCoverage", CoverageLevel::getDisplayColorsOfCoverageLevel),
     /**
      * Difference between the project coverage and the modified lines coverage of the current build. Teams can use this delta
      * value to ensure that the coverage of pull requests is better than the whole project coverage.
      */
-    MODIFIED_LINES_DELTA(Messages._Baseline_MODIFIED_LINES_DELTA(), "changeCoverage",
+    MODIFIED_LINES_DELTA(Messages._Baseline_MODIFIED_LINES_DELTA(), "modifiedLinesCoverage",
             CoverageChangeTendency::getDisplayColorsForTendency),
     /**
      * Coverage of the modified files (e.g., within the files that have been touched in a pull or merge request) will
      * focus on new or modified code only.
      */
-    MODIFIED_FILES(Messages._Baseline_MODIFIED_FILES(), "fileCoverage", CoverageLevel::getDisplayColorsOfCoverageLevel),
+    MODIFIED_FILES(Messages._Baseline_MODIFIED_FILES(), "modifiedFilesCoverage", CoverageLevel::getDisplayColorsOfCoverageLevel),
     /**
      * Difference between the project coverage and the modified file coverage of the current build. Teams can use this delta
      * value to ensure that the coverage of pull requests is better than the whole project coverage.
      */
-    MODIFIED_FILES_DELTA(Messages._Baseline_MODIFIED_FILES_DELTA(), "fileCoverage", CoverageChangeTendency::getDisplayColorsForTendency),
+    MODIFIED_FILES_DELTA(Messages._Baseline_MODIFIED_FILES_DELTA(), "modifiedFilesCoverage", CoverageChangeTendency::getDisplayColorsForTendency),
```

Comment:
While looking at the results I think I made a mistake here. When I am developing a PR normally I am interested in the coverage of my changes. 

That means we need the totals (this is already working correctly): 
- coverage overall project 
- changed files
- changed lines

But when looking at the delta we need:
- delta overall project - delta overall reference (typically not interesting for large projects)
- delta changed files - delta changed files reference (how is the coverage of the changed files affected)? Currently we check how it changes with respect to the overall coverage.
- delta changed lines: here I am not sure whether it makes more sense to compare vs. the whole project or the changed files. What do you think? Or do we need both?

Reply:
Yes, that makes sense. 
Regarding the delta changed lines: I guess we should compare only against the modified files coverage. The whole project has modules with higher and with lower coverage. When we compare the changes of a module with lower coverage against the whole project, there will be a negative trend, even if the modified files are covered quiet good compared to the rest of their module. Therefore, comparing against the changed files would keep the context hence it more accurate, in my opinion.